seed/tests/test_salesforce_views.py

```diff
@@ -356,3 +352,260 @@ def test_no_sync_when_disabled(self):
         self.assertIn('Salesforce Workflow is not enabled', data['message'][0])
 
     # TODO: test auto sync works and sets date
+
+
+class SalesforceViewTestPermissions(AccessLevelBaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.api_view = LabelInventoryViewSet()
+        self.cycle = self.cycle_factory.get_cycle(
+            start=datetime(2010, 10, 10, tzinfo=get_current_timezone()))
+        self.column_list_factory = FakeColumnListProfileFactory(organization=self.org)
+        self.login_as_root_member()
+
+        # setup labels
+        self.ind_label = Label.objects.create(
+            name='Indication Label', super_organization=self.org
+        )
+        self.violation_label = Label.objects.create(
+            name='Violation Label', super_organization=self.org
+        )
+        self.compliance_label = Label.objects.create(
+            name='Compliance Label', super_organization=self.org
+        )
```

Comment:
these labels were also created in `setUp()`

Reply:
@haneslinger  sorry, not sure what you mean here...this is the setUp function